src/Symfony/Component/AssetMapper/Tests/ImportMap/ImportMapManagerTest.php

```diff
@@ -373,6 +373,65 @@ public function testUpdate()
         $this->assertSame('contents of cowsay.js', $actualContents);
     }
 
+    public function testUpdateWithSpecificPackages()
+    {
+        $rootDir = __DIR__.'/../fixtures/importmaps_for_writing';
+        $manager = $this->createImportMapManager(['assets' => ''], $rootDir);
+
+        $map = [
+            'lodash' => [
+                'url' => 'https://ga.jspm.io/npm:lodash@1.2.3/lodash.js',
+            ],
+            'cowsay' => [
+                'url' => 'https://ga.jspm.io/npm:cowsay@4.5.6/cowsay.umd.js',
+                'downloaded_to' => 'vendor/moo.js',
```

Comment:
I can see you copied this from another test, but it'd be simpler have it be `vendor/cowsay.js`